freeipa/src/main/java/com/sequenceiq/freeipa/entity/Stack.java

```diff
@@ -48,6 +50,9 @@
 
     private String environmentCrn;
 
+    @OneToMany(mappedBy = "stack", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<ChildEnvironment> childEnvironments = new ArrayList<>();
```

Comment:
do we really need it eagerly? this would mean an extra join every time we fetch the stack

Reply:
Seems like we do not really need it, I have a PR for this alone
https://github.com/hortonworks/cloudbreak/pull/7288